NAMESPACE

```diff
@@ -126,6 +126,7 @@ export(GeomAbline)
 export(GeomAnnotationMap)
 export(GeomArea)
 export(GeomBar)
+export(GeomBind2d)
```

Comment:
typo?

Reply:
Since I had to create a new class, I had to export this as well.

Reply:
Let me fix this too!